buildSrc/DesktopBuilder.js

```diff
@@ -52,24 +51,28 @@ function build(dirname, version, targets, updateUrl, nameSuffix, notarize) {
 			})
 		})
 		.then(() => {
-			console.log("Move output to /build/" + updateSubDir + "/...")
+			const installerDir = path.join(distDir, 'installers')
+			console.log("Move artifacts to", outDir)
+			const unpackedFilter = file => file.endsWith("-unpacked") || file === "mac"
```

Comment:
I'm not sure I understand this and it also looks kinda brittle

Reply:
Since the names of the artifacts are highly inconsistent, they differ between platforms, targets and release stages. That filter function prevents us from having to define a list with all the possible artifact names that need to be moved, or one that excludes everything we _don't_ want.

The directory the artifacts are in is created for the build, so there should not be anything in there that should not be copied to the output directory.

It is brittle, but not much more so than a list of artifact names, I reckon.

I'll see if I can simplify it by deleting everything we're not interested in (atm that's one file I think) before moving the rest and add an explanation.